design-details/sensing.tex

```diff
@@ -6,65 +6,101 @@ \subsection{Sensing Subsystem}
 
-That’s right, the first part of the optical subsystem is the soil. Structures in the soil emit trace electromagnetic signals that correspond to the soil matrix, soil temperature, soil moisture, and chemical content of the soil. (Link papers)
-In order to ensure sufficient signal to noise ratio, the dirt is going to have to be probed by a bright, broadband source covering the visible and Near Infrared regime. Halogen lamps will do (link papers) but it may be possible to achieve the same effect with LEDs near the surface. 
-An additional concern is that irregularities in the soil topography may reflect back light that would otherwise indicate organic components or water. In order to counter this, it is going to be necessary to flatten the dirt with a stamp or roller.
+\begin{figure}[H]
+    \caption{Spectral Output of a Tungsten Lamp}
+    \centering
+    \includegraphics[width=0.75\textwidth]{images/TungstenLamp.jpg}
+\end{figure}
+Reference Link: https://zeiss-campus.magnet.fsu.edu/articles/lightsources/tungstenhalogen.html
 
+\paragraph{Soil Conditions} The optical signal emitted by the soil will be weak, so it is essential that the spectrometer is influenced as much as possible by the content of the soil and as little as possible by the topology of the surface. A well-mixed, smoothly flattened, gently compressed soil sample provides the best conditions for constructing a spectrograph. If there are ridges or depressions along the sample, too much or too little optical signal will pass into the spectrometer, creating an apparent signal strength for wavelengths that is not representative of the soil emission. There are several ways to control these conditions. One is to require the user to retrieve a sample for each scan and deposit it into a spectrometer bay. Another is to require the user to flatten the soil with a spatula or other implement and position the scanner input near the soil. These reduce ease of use, which is a target for the project. Another issue is that the position of the signal input and the tungsten light source relative to the soil sample will have to be consistent to avoid signal error, for the same reasons as above. In (reference) study, this issue was resolved by building a transparent block with slots to hold the light source and the input beam. Their design is represented below, showing the block attached at the rear of a plow chisel for clearing soil while scanning. We will solve the problem of soil topology and light position the same way, by mounting the light source and input fiber in a block of some transparent material like acrylic.
 
-\paragraph{LEDs}
+\begin{figure}[H]
+    \caption{Fiber Position in Soil Drill}
+    \centering
+    \includegraphics[width=0.75\textwidth]{images/Acrylic Block Fiber positioner.png}
+\end{figure}
+Reference paper titled: On line measurement of some selected soil properties using a VIS NIR sensor
 
```

Comment:
Remove references like this and just send me the link or reference formatted into title, author, date, etc.